fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchTest.java

```diff
@@ -553,6 +554,51 @@ public void testSearchPatientWithIdentifierOfTypeNotFound() {
         assertTrue(bundle.getEntry().isEmpty());
     }
 
+    @Test(groups = { "server-search" }, dependsOnMethods = {"testCreatePatient" })
+    public void testSearchPatientInvalidSearchParameter_lenient() {
+        WebTarget target = getWebTarget();
+        Response response =
+                target.path("Patient").queryParam("unknown", "unknown").request(FHIRMediaType.APPLICATION_FHIR_JSON)
+                .header("Prefer", "handling=lenient")
+                .get();
+        assertResponse(response, Response.Status.OK.getStatusCode());
+        Bundle bundle = response.readEntity(Bundle.class);
+
+        assertNotNull(bundle);
+        assertTrue(bundle.getEntry().size() >= 1);
+        assertTrue(SearchEntryMode.of(bundle.getEntry().get(bundle.getEntry().size()-1).getSearch().getMode().getValue()) == SearchEntryMode.OUTCOME);
```

Comment:
I think either way would work, but I prefer the following:
```suggestion
        assertTrue(bundle.getEntry().get(bundle.getEntry().size()-1).getSearch().getMode().getValueAsEnum()) == SearchEntryMode.ValueSet.OUTCOME);
```

Reply:
Updated.